static/css/style.css

```diff
@@ -32,6 +39,65 @@ p {
   color: #172b4d;
 }
 
+button {
+  padding: 0.75em;
+  background: #36b37f;
+  color: #fff;
+  letter-spacing: 0.03em;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  border: 0;
+  border-radius: 0.25em;
+  min-width: 9.93em;
+  min-height: 3em;
+}
+
+button:hover { box-shadow: 5px 0.5em 1em rgba(54, 179, 127, 0.24); }
+
+button:active { background-color: #008552; }
```

Comment:
- [ ] great job on styling the button 👍 I do however recommend adding the cursor: pointer to the styling so it will be more intuitive for the users.